service/resource/configmap/current.go

```diff
@@ -27,5 +27,8 @@ func (r *Resource) GetCurrentState(ctx context.Context, obj interface{}) (interf
 	}
 
 	r.logger.Log("debug", "found configmap")
+
+	configmapSize.Set(float64(configMap.Size()))
+
```

Comment:
For my interest. Is it growing over time?

Reply:
I guess the metrics will tell us. :p

Reply:
Currently, it will be growing over time (we don't remove services from the configmap). When we do remove services, it should be stable over time.